tensorflow-app-dependencies/pom.xml

```diff
@@ -0,0 +1,125 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.springframework.cloud.stream.app</groupId>
+    <artifactId>tensorflow-app-dependencies</artifactId>
+    <version>1.2.1.BUILD-SNAPSHOT</version>
+    <packaging>pom</packaging>
+    <name>tensorflow-app-dependencies</name>
+    <description>Spring Cloud Stream tensorflow App Dependencies</description>
+
+    <parent>
+		<artifactId>spring-cloud-dependencies-parent</artifactId>
+        <groupId>org.springframework.cloud</groupId>
+        <version>1.2.1.RELEASE</version>
+        <relativePath />
+    </parent>
+
+    <properties>
+        <pmml.version>1.2.6</pmml.version>
+		<tensorflow.version>1.1.0-rc2</tensorflow.version>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+			<dependency>
+				<groupId>org.springframework.cloud</groupId>
+				<artifactId>spring-cloud-bus</artifactId>
```

Comment:
This is to allow hot-swap of TensorFlow models without restarting the processor or the pipeline. See: https://github.com/tzolov/tensorflow-spring-cloud-stream-app-starters/issues/2
Still need to document the hwo auto refresh configuration somewhere i guess.